File: uikit/src/main/java/screen/adminAndModeratorList/CometChatAdminModeratorListScreenActivity.java
package screen.adminAndModeratorList;

import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import com.cometchat.pro.uikit.R;

import constant.StringContract;

public class CometChatAdminModeratorListScreenActivity extends AppCompatActivity {


    private String guid;

    private String ownerId;

    private boolean showModerator;

    private String loggedInUserScope;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_block_screen);
        handleIntent();
    }

    private void handleIntent() {
        if (getIntent().hasExtra(StringContract.IntentStrings.MEMBER_SCOPE)){
            loggedInUserScope=getIntent().getStringExtra(StringContract.IntentStrings.MEMBER_SCOPE);
        }
        if (getIntent().hasExtra(StringContract.IntentStrings.GUID)) {
            guid = getIntent().getStringExtra(StringContract.IntentStrings.GUID);
        }
        if (getIntent().hasExtra(StringContract.IntentStrings.GROUP_OWNER)) {
            ownerId = getIntent().getStringExtra(StringContract.IntentStrings.GROUP_OWNER);
        }
        if (getIntent().hasExtra(StringContract.IntentStrings.SHOW_MODERATORLIST)) {
            showModerator = getIntent().getBooleanExtra(StringContract.IntentStrings.SHOW_MODERATORLIST,false);
        }
        Fragment fragment = new CometChatAdminModeratorListScreen();
        Bundle bundle = new Bundle();
        bundle.putString(StringContract.IntentStrings.GUID,guid);
        bundle.putString(StringContract.IntentStrings.GROUP_OWNER,ownerId);
        bundle.putString(StringContract.IntentStrings.MEMBER_SCOPE,loggedInUserScope);
        bundle.putBoolean(StringContract.IntentStrings.SHOW_MODERATORLIST,showModerator);
        fragment.setArguments(bundle);
        getSupportFragmentManager().beginTransaction().replace(R.id.frame_fragment,fragment).commit();

    }


    @Override
    public void onBackPressed() {
        super.onBackPressed();
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        return super.onCreateOptionsMenu(menu);
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {

           if(item.getItemId()==android.R.id.home){
               onBackPressed();
           }

        return super.onOptionsItemSelected(item);
    }


}

File: uikit-kotlin/src/main/java/screen/adminAndModeratorList/CometChatAdminModeratorListScreenActivity.kt
package screen.adminAndModeratorList

import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.cometchat.pro.uikit.R
import constant.StringContract

class CometChatAdminModeratorListScreenActivity : AppCompatActivity() {
    private var guid: String? = null
    private var ownerId: String? = null
    private var showModerator = false
    private var loggedInUserScope: String? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_block_screen)
        handleIntent()
    }

    private fun handleIntent() {
        if (intent.hasExtra(StringContract.IntentStrings.MEMBER_SCOPE)) {
            loggedInUserScope = intent.getStringExtra(StringContract.IntentStrings.MEMBER_SCOPE)
        }
        if (intent.hasExtra(StringContract.IntentStrings.GUID)) {
            guid = intent.getStringExtra(StringContract.IntentStrings.GUID)
        }
        if (intent.hasExtra(StringContract.IntentStrings.GROUP_OWNER)) {
            ownerId = intent.getStringExtra(StringContract.IntentStrings.GROUP_OWNER)
        }
        if (intent.hasExtra(StringContract.IntentStrings.SHOW_MODERATORLIST)) {
            showModerator = intent.getBooleanExtra(StringContract.IntentStrings.SHOW_MODERATORLIST, false)
        }
        val fragment: Fragment = CometChatAdminModeratorListScreen()
        val bundle = Bundle()
        bundle.putString(StringContract.IntentStrings.GUID, guid)
        bundle.putString(StringContract.IntentStrings.GROUP_OWNER, ownerId)
        bundle.putString(StringContract.IntentStrings.MEMBER_SCOPE, loggedInUserScope)
        bundle.putBoolean(StringContract.IntentStrings.SHOW_MODERATORLIST, showModerator)
        fragment.arguments = bundle
        supportFragmentManager.beginTransaction().replace(R.id.frame_fragment, fragment).commit()
    }

    override fun onBackPressed() {
        super.onBackPressed()
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            onBackPressed()
        }
        return super.onOptionsItemSelected(item)
    }
}